media-gfx/blender/blender-2.83.13.ebuild

```diff
@@ -247,17 +285,27 @@ src_compile() {
 }
 
 src_test() {
-	if use test; then
-		einfo "Running Blender Unit Tests ..."
-		cd "${BUILD_DIR}"/bin/tests || die
-		local f
-		for f in *_test; do
-			./"${f}" || die
-		done
-	fi
+	# A lot of tests needs to have access to the installed data files.
+	# So install them into the image directory now.
+	cmake_src_install
```

Comment:
Will this work? The install phase expects the test phase to be finished and probably checks for the existence of `${WORKDIR}/.tested`, which isn't set before sucessfully finishing the test phase.

Reply:
I double checked with the live ebuild and it still seems to work fine for me.
So it seems to work on my end.